addons/mail/static/src/messaging/component/activity/activity.xml

```diff
@@ -110,7 +110,7 @@
                     </t>
 
                     <t t-if="activity.canWrite">
-                        <div class="o_Activity_tools">
+                        <div name="mail_activity_tools" class="o_Activity_tools">
```

Comment:
why not just `tools` for name?
Idea is that name has the same semantic as the part of the component, which here is `tools`.

Reply:
Didn't know the guideline so I tried something ahah, will change :)